### PR TITLE
Release only non-empty sessions

### PR DESCRIPTION
**Pull request by dmdashenkov**
Before this PR, when releasing shard processing sessions, we cleaned up all the sessions regardless of whether or not a session is active (has a processing node) or not. Now, we only release the active sessions.

See the [same change](https://github.com/SpineEventEngine/core-java/pull/1157) in `core-java`.

### CODE REVIEW

**Comment by dmdashenkov** [comment]
@alexander-yevsyukov, PTAL.

**Comment by dmdashenkov** [comment]
@alexander-yevsyukov, PTAL again.

File: version.gradle [hunk 1]
```diff
@@ -25,12 +25,12 @@
...
  */
 
-def final SPINE_VERSION = '1.0.3'
+def final SPINE_VERSION = '1.0.7-SNAPSHOT'
 
 ext {
     versionToPublish = SPINE_VERSION
     
-    spineBaseVersion = SPINE_VERSION
+    spineBaseVersion = '1.0.3'
```
Comment (alexander-yevsyukov) [inline version.gradle]: Please use the recent version of `base`.

**Review by alexander-yevsyukov — APPROVED** [review]
LGTM with one comment related to the version of `base`.

**Pull request merged**